ocs_ci/utility/utils.py

```diff
@@ -5041,3 +5040,32 @@ def sum_of_two_storage_sizes(storage_size1, storage_size2, convert_size=1024):
     size = size1 + size2
     new_storage_size = f"{size}{unit}"
     return new_storage_size
+
+
+class CustomJSONEncoder(json.JSONEncoder):
+    """
+    Custom JSON encoder to handle set objects
+    """
+
+    def default(self, obj):
+        if isinstance(obj, set):
+            return list(obj)
+        return super().default(obj)
+
+
+def get_odf_tag_from_redhat_catsrc():
```

Comment:
It would be better to have this function in ocs/resources/catalog_source.py.

Reply:
moved, thanks